server/src/main/java/server/cache/BuildTarget.java

```diff
@@ -0,0 +1,27 @@
+package server.cache;
+
+import java.nio.file.Path;
+
+public class BuildTarget {
```

Comment:
This looks like an object that the Bazel package should also know about. E.g. when we call `BazelInvoker.getBuildTargets(...)`, we might want to return this type instead of duplicating types. Maybe we ought to move this `server.cache` package to be the actual `server.bazel` or possibly `server.bazel.domain` package?